### PR TITLE
[interp] Avoid declaring MonoError's in interp_exec_method_full (), it increases the native stack frame size because the optimizer can't seem to figure out live ranges.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by lewurm** [comment]
does that help with https://github.com/mono/mono/issues/14611 ?

**Comment by vargaz** [comment]
No, but it should increase perf a little bit, and prevent possible stack overflows on some platforms like wasm.

File: mono/mini/interp/interp.c [hunk 1]
```diff
@@ -992,7 +991,7 @@ ves_array_calculate_index (MonoArray *ao, stackval *sp, InterpFrame *frame, gboo
 	return pos;
 }
 
-static void
+static MONO_NEVER_INLINE void
```
Comment (jaykrell) [inline mono/mini/interp/interp.c]: These might merit
comments — “to avoid growing caller’s frame” or somesuch.
NT does similar.

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

**Pull request merged**